### PR TITLE
About page

### LINKED ISSUE #33: How it works page
## Description

Users can see how to use Shroomy, the smart shopping list. Each section describes how to navigate the app and has corresponding images to show what to expect.

## Related Issue

Closes #33

## Acceptance Criteria

- [ ] Descriptions accurately explain how the app works
- [ ] Each feature description has a corresponding image to show that feature
- [ ] The page scrolls until the end of the content

### PR DESCRIPTION
**Pull request by kurzbird**
## Description

Users can see how to use Shroomy, the smart shopping list. Each section describes how to navigate the app and has corresponding images to show what to expect.

## Related Issue

Closes #33

## Acceptance Criteria

- [x] Descriptions accurately explain how the app works
- [x] Each feature description has a corresponding image to show that feature
- [x] The page scrolls until the end of the content

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before
Screenshot 2023-03-01 at 10 07 37 AM

### After

image

image

image

image

### CODE REVIEW

**Review by jennaly — APPROVED** [review]
Great UI and copywriting! Depending on how much padding/margin there is at the bottom of the header, you may want to add margin at the top of your component to give them some space.

**Comment by neshacascia** [comment]
Love how the about page turned out!

File: src/views/AddItem.jsx [hunk 1]
```diff
@@ -2,15 +2,17 @@ import { useState } from 'react';
...
 
 	const SOON = 7;
 	const KINDA_SOON = 14;
 	const NOT_SOON = 30;
 
 	const [itemName, setItemName] = useState('');
 	const [nextPurchase, setNextPurchase] = useState(0);
-	const [submissionConfirmation, setSubmissionConfirmation] = useState('');
+	const [submissionConfirmation, setSubmissionConfirmation] = useState(
+		'TO BE REMOVED: placeholder for confirmation message',
```
Comment (spittel) [inline src/views/AddItem.jsx:14]: Did you mean to leave this placeholder here?

File: src/views/AddItem.jsx [hunk 2]
```diff
@@ -76,57 +78,86 @@ export function AddItem({ data, listToken }) {
...
+				>
+					<h1 className="font-bold uppercase text-center my-5">add new item</h1>
+					<label htmlFor="itemName" className="text-center">
+						Item Name
+					</label>
 					<input
 						type="text"
 						id="itemName"
-						value={itemName}
+						value={itemName ? itemName : undefined}
```
Comment (spittel) [inline src/views/AddItem.jsx:97]: Are we displaying 'undefined' to the user if there is no itemName? I'm not sure if that's the most user-friendly message?

File: src/views/AddItem.jsx [hunk 3]
```diff
@@ -76,57 +78,86 @@ export function AddItem({ data, listToken }) {
...
-						/>
-						Not Soon
-					</label>
-				</fieldset>
-
-				<button type="submit">Add Item</button>
-			</form>
 
-			{submissionConfirmation && <p>{submissionConfirmation}</p>}
+					{/* <label htmlFor="buyAgain">How soon will you buy this again?</label> */}
```
Comment (spittel) [inline src/views/AddItem.jsx:103]: Nuke the dead code! :)

File: src/views/Layout.css [hunk 4]
```diff
@@ -7,7 +7,7 @@
  *
  * @see: https://developer.mozilla.org/en-US/docs/Web/CSS/env
  */
-.Layout {
+/* .Layout {
```
Comment (spittel) [inline src/views/Layout.css]: Seen this on other PRs, vast swaths of code and CSS that have been commented out. Just delete it, YAGNI

File: src/views/List.jsx [hunk 5]
```diff
@@ -24,6 +24,9 @@ export function List({ data, listToken }) {
 
 	return (
 		<>
+			{/* The height of the green section of the navigation tab is: h-20 (5rem);
+		The button is sticking out by -mt-6 (1.5rem);
+		You'll need 6.5rem of empty space at the bottom of your page so that the user can see their entire list.  */}
```
Comment (spittel) [inline src/views/List.jsx]: Is this a dead comment? Or a comment for a different dev?

**Review by spittel — APPROVED** [review]
Some questions about dead code. But overall looks great!

File: src/views/List.jsx (hunk 5 above)
Comment (FuentesNina) [reply src/views/List.jsx]: @spittel The deadcode on this PR is from code outside of the scope of this PR [the code you saw were outdated]. But you're right, we do have a bunch of commented out CSS that we need to deleted. I'm working on that and it should be gone when we merge.

**Pull request merged**